int16 Reffbox
int16 Umpan
int16 Kondisi
int16 Ballposx
int16 Ballposy
int8 Balldetect
int16 ModeRun
int16 xtarget
int16 ytarget
int16 ttarget

int16 memberX
int16 memberY
int16 memberT

int16 kiperX
int16 kiperY
int16 kiperT

int16 obstacleX1
int16 obstacleY1
int16 obstacleX2
int16 obstacleY2
int16 obstacleX3
int16 obstacleY3

int8 flagCalib
int16 kalibX
int16 kalibY
int16 kalibH

int8 hendroDetect



 